guides/common/modules/proc_exporting-templates.adoc

```diff
@@ -14,25 +17,51 @@ For more information about each field, see xref:Configuring_the_TemplateSync_plu
 The {ProjectWebUI} displays the status of the export.
 The status is not persistent; if you leave the status page, you cannot return to it.
 
+[id="cli_Exporting_Templates_{context}"]
 .CLI procedure
-. Clone a local copy of your Git repository:
+. To export the templates to a repository, enter the following command:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-$ git clone https://github.com/theforeman/community-templates __/custom/templates__
+hammer export-templates \
+--organization "Default Organization" \
+--repo "_https://git.example.com/path/to/repository_"
 ----
-. Change the owner of your local directory to the `foreman` user, and change the SELinux context with the following commands:
 +
-[options="nowrap", subs="+quotes,attributes"]
+[NOTE]
+====
+This command clones the repository, makes changes in a commit, and pushes back to the repository.
+You can use the `--branch "_My_Branch_"` option to export the templates to a specific branch.
+====
+
+[id="api_Exporting_Templates_{context}"]
+.API Procedure
+
+. Send a `POST` request to `api/v2/templates/export`:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
 ----
-# chown -R foreman:foreman __/custom/templates__
-# chcon -R -t httpd_sys_rw_content_t __/custom/templates__
+# curl -H "Accept:application/json" \
+-H "Content-Type:application/json" \
+-u _login_:__password__ \
+-k https://_{foreman-example-com}/api/v2/templates/export \
+-X POST
 ----
-. To export the templates to your local repository, enter the following command:
 +
-[options="nowrap", subs="+quotes,attributes"]
+If the export is successful, you receive `{"message":"Success"}`.
+
+[NOTE]
+====
+You can override default API settings by specifying them in the request with the `-d` parameter.
+The following example exports templates to the `git.example.com/templates` repository:
```

Comment:
```suggestion
The following example exports templates to the `_git.example.com_/templates` repository:
```

Reply:
The whole code block is presented as an example so in this case I will leave it as is.